### PR TITLE
[UI enhancment] add sticky navbar in mobile views

### PR DESCRIPTION
**Pull request by restylianos**
relates to #388

### CODE REVIEW

**Comment by fliebenberg** [comment]
@EvgeniiaVak What does it mean when it says "Waiting for status to be reported" for the Cloudflare Pages. I am not sure if there is something I can do to progress this PR.

**Comment by dcts** [comment]
@fliebenberg we don't have deploy previews setup for PRs from forked repos. 
I copied the branch and uploaded to the main repo to get a deploy preview, you can find it here:
https://restylianos-sticky-nav.dexter-1we.pages.dev/

File: src/app/components/NavBar.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const NavItems: { path: string; title: string }[] = [
 
 export function Navbar() {
   return (
-    <nav className="flex items-center justify-between w-full !h-[74px] !min-h-[74px]">
+    <nav className="sticky top-0 sm:static z-50 bg-base-300 flex items-center justify-between w-full !h-[74px] !min-h-[74px]">
```
Comment (dcts) [inline src/app/components/NavBar.tsx:36]: I believe the color should be base-200 (grey) not base-300 (black)
```suggestion
    <nav className="sticky top-0 sm:static z-50 bg-base-200 flex items-center justify-between w-full !h-[74px] !min-h-[74px]">
```

**Comment by dcts** [comment]
@restylianos I see that this PR is from your fork's main branch. While it doesn't create any issues for the original repo (https://github.com/DeXter-on-Radix/website) it might make it more difficult for you to keep your fork up to date. I advise you to create feature branches for PRs.

(fun fact: I did the exact same thing when I submitted my first PR in February 2024, so I copy&pasted @EvgeniiaVak 's response here, for reference: https://github.com/DeXter-on-Radix/website/pull/242#issuecomment-1921936230)

**Review by dcts — CHANGES_REQUESTED** [review]
Our designer @maurojose also suggested to add a thin grey bottom border on the sticky nav (only on mobile) for a better separation between content / navbar when scrolling. 

I suggest to add something like `border-b border-gray-800` for example (only for mobile).

**Review by dcts — DISMISSED** [review]
Thanks for implementing! 

Deplyo preview is here if any other reviewer wants to check:
https://restylianos-sticky-nav.dexter-1we.pages.dev/

**Review by fliebenberg — APPROVED** [review]
Sticky Navbar working great. Thanks.

**Comment by fliebenberg** [comment]
I have now also approved. Not sure what we need to do next to be able to merge this PR.

**Comment by dcts** [comment]
@EvgeniiaVak do you know how we can manually trigger the claoudflare pages requirement to merge this PR? I think the issue is that the PR comes from a forked repo but not sure how to handle this.

**Comment by dcts** [comment]
@restylianos please copy this PR as well and create a new branch on the original repo, similar to what you did in #408 :

```
# 1. Clone the original repository (if you haven't already)
git clone https://github.com/dexter-on-radix/website.git dexter-webapp
cd dexter-webapp

# 2. Add your fork as a remote (if you haven't already):
git remote add fork-restylianos https://github.com/restylianos/website.git

# 3. Fetch the branches from your fork:
git fetch fork-restylianos

# 4. Create a new branch in the original repository and copy the PR from your forked repository's branch:
git checkout -b fork-restylianos--main fork-restylianos/main

# 5. Push the new branch to the original repository:
git push origin fork-restylianos--main

# Create a pull request for 'fork-restylianos--main' on GitHub by visiting:
#      https://github.com/DeXter-on-Radix/website/pull/new/fork-restylianos--main
```

**Pull request closed without merging**